src/main/webapp/css/style.css

```diff
@@ -22,3 +22,32 @@ h1 {
 .green {
   color: #0F9D58;
 }
+
+.header-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between; 
+  margin-left: 48%;
```

Comment:
I am very suspicious of this layout :/ but as requested, I'll delay the rest of my judgment to the PR that changes all this CSS.